### PR TITLE
Updated to RunDec3.1

### PR DESCRIPTION
**Pull request by mfael86**
CRunDec has been upgraded to version 3.1. It now includes routines for the conversion to the kinetic mass.
Arguments declared in the c++ code as `std::pair<double,double>* mq` can be passed with class `rundec.PairDouble()`

### CODE REVIEW

**Comment by DavidMStraub** [comment]
Thanks a lot Matteo! I haven't had time to test it yet; would you be willing to add a simple unit test? Then we could set up continuous integration, which might also be helpful for future changes.

**Comment by mfael86** [comment]
Hi David, ok I will prepare the unit test to cross check the results with Mathematica.

**Comment by mfael86** [comment]
Unittest prepared and the branch ready to merge. Masses of light quarks in functions for mass conversion can be passed via ```rundec.RunDecPair()``` and ```rundec.RunDecPairArray(4)```. Masses of light quarks in functions for decoupling relation can be passed via ```rundec.TriplenfMmu()``` and ```rundec.TriplenfMmuArray(4)```.

**Comment by DavidMStraub** [comment]
Thanks Matteo!

Please add `extras_require={'testing': ['pytest']}` to `setup.py`.

**Comment by mfael86** [comment]
I updated ```setup.py```

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
       - name: Install Python dependencies
         run: |
           pip install .[testing]
+          pip install pytest
```
Comment (DavidMStraub) [inline .github/workflows/test.yml:24]: Now that pytest is in the `testing` extra requirement, this line is redudant btw. But doesn't hurt either.

**Comment by DavidMStraub** [comment]
Congrats on your contribution 🙂

**Pull request merged**